modules/apps/account/account-service/service.xml

```diff
@@ -115,6 +115,33 @@
 		<reference entity="AccountEntry" package-path="com.liferay.account" />
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity external-reference-code="true" local-service="true" name="AccountGroup" remote-service="true">
+
+		<!-- PK fields -->
+
+		<column name="accountGroupId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="createDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="name" type="String" />
+		<column name="description" type="String" />
+
+		<!-- Order -->
+
+		<order by="asc">
+			<order-column name="name" />
+		</order>
+
+		<!-- References -->
+
+		<reference entity="User" package-path="com.liferay.portal" />
+	</entity>
```

Comment:
Should we use the same fields that commerce has for AccountGroup? https://github.com/liferay/com-liferay-commerce/blob/7.1.x/commerce-account-service/service.xml#L62-L105

Getting all the fields needed will help us avoid writing upgrade processes later. What do you think @drewbrokke?

Reply:
I originally advised him to include only the needed fields, but I see your point about not writing upgrade processes later. It won't do any harm to add extra fields, and considering Commerce would like to be able to migrate at some point, it makes sense to provide them what the need at this stage.

Adding all the fields is fine by me :)

Reply:
Sounds good. I'll run by Dennis to make sure we want all the fields.